banyand/measure/merger_algo.go

```diff
@@ -0,0 +1,133 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
```

Comment:
How about renaming the file to `merge_policy.go` since there is only a struct with the same name in it?